build.rs

```diff
@@ -305,37 +306,14 @@ fn build_c_code(out_dir: PathBuf) {
 
 fn build_msvc(target_info: &TargetInfo, disable_opt: bool, num_jobs: &str,
               lib_path: PathBuf) {
-    let (platform, optional_amd64) = match target_info.target_arch() {
-        "x86" => ("Win32", None),
-        "x86_64" => ("x64", Some("amd64")),
+    use gcc::windows_registry;
```

Comment:
This seems reasonable, but why is it part of this PR? I think we should leave the Windows build alone for now.